shallow_backup/cli.py

```diff
@@ -29,14 +29,15 @@
 @click.option('-reinstall_packages', is_flag=True, default=False, help="Reinstall packages.")
 @click.option('-reinstall_all', is_flag=True, default=False, help="Full reinstallation.")
 @click.option('--remote', default=None, help="Set remote URL for the git repo.")
-@click.option('--rm', default=None, type=str, help="Remove path from config.")
-@click.option('-show', is_flag=True, default=False, help="Show config file.")
+@click.option('--rm', default=None, type=str, help="Remove path from backup.")
```

Comment:
Add note about how to remove configs and plists (and check that it's actually implemented too)